File: File Upload/Generate Secure URL for Upload/JavaScript/GenerateSecureUrlForUpload.js
//*******************************************************************************************//
//                                                                                           //
// Download Free Evaluation Version From: https://bytescout.com/download/web-installer       //
//                                                                                           //
// Also available as Web API! Get Your Free API Key: https://app.pdf.co/signup               //
//                                                                                           //
// Copyright © 2017-2020 ByteScout, Inc. All rights reserved.                                //
// https://www.bytescout.com                                                                 //
// https://pdf.co                                                                            //
//                                                                                           //
//*******************************************************************************************//


var request = require('request');
var options = {
  'method': 'GET',
  'url': 'https://api.pdf.co/v1/file/upload/get-presigned-url?name=test.pdf&encrypt=true',
  'headers': {
    'x-api-key': '{{x-api-key}}'
  },
  formData: {

  }
};
request(options, function (error, response) {
  if (error) throw new Error(error);
  console.log(response.body);
});

File: File Upload/Generate Secure URL for Upload/Python/GenerateSecureUrlForUpload.py
import requests

url = "https://api.pdf.co/v1/file/upload/get-presigned-url?name=test.pdf&encrypt=true"

payload = {}
files = {}
headers = {
		'x-api-key': '{{x-api-key}}'
}

response = requests.request("GET", url, headers=headers, data = payload, files = files)

print(response.text.encode('utf8'))
